server/service/src/activity_log.rs

```diff
@@ -51,6 +51,39 @@ pub fn activity_log_entry(
         },
         record_id: Some(record_id.to_string()),
         datetime: Utc::now().naive_utc(),
+        event: None,
+    };
+
+    Ok(ActivityLogRowRepository::new(&ctx.connection).insert_one(log)?)
+}
+
+pub fn activity_log_stock_entry(
+    ctx: &ServiceContext,
+    log_type: ActivityLogType,
+    record_id: &str,
+    from: Option<String>,
+    to: Option<String>,
+) -> Result<(), RepositoryError> {
+    let log = &ActivityLogRow {
+        id: uuid(),
+        r#type: log_type,
+        user_id: if ctx.user_id != "" {
+            Some(ctx.user_id.clone())
+        } else {
+            None
+        },
+        store_id: if ctx.store_id != "" {
+            Some(ctx.store_id.clone())
+        } else {
+            None
+        },
+        record_id: Some(record_id.to_string()),
+        datetime: Utc::now().naive_utc(),
+        event: Some(format!(
+            "Changed from {} to {}",
```

Comment:
more of a thinking aloud than a suggestion. Wondering if putting something around the values would make it easier for people to read:
```suggestion
            "Changed from [{}] to [{}]",
```